output/schema/schema.json

```
@@ -110992,6 +110992,24 @@
       },
       "specLocation": "eql/search/types.ts#L20-L32"
     },
+    {
+      "kind": "enum",
+      "members": [
+        {
+          "description": "Run against the first version of ESQL.",
```

super nit, but we could be more consistent with our use of ESQL vs ES|QL:
```suggestion
          "description": "Run against the first version of ES|QL.",
```
Also applies below and in `Version.ts`.